app/server/src/lib/compiler/stringifier.ts

```diff
@@ -5,7 +5,7 @@ const { identifier } = require('safe-identifier');
 
 export function stringifyObject(object: any, level = 0, excludeOuterParams = false): string {
   if (typeof object === 'string') {
-    return `'${object}'`;
+    return `"${object}"`;
```

Comment:
Won't this version have the same problem as the original but in the case of double quotes? I suggest you use `JSON.stringify` (or something similar) that handles string escaping for you. WDYT?

Reply:
Great suggestion.  Updated to do that.